src/modules/75-cf/pages/feature-flags/components/FeatureFlagsListing.tsx

```diff
@@ -102,6 +117,33 @@ const FeatureFlagsListing: FC<FeatureFlagsListingProps> = ({
 
   const columns: Column<Feature>[] = useMemo(
     () => [
+      {
+        id: 'staleCheckbox',
+        Header: (
+          <Checkbox
+            data-testid={'selectAll'}
+            onClick={() => {
+              //TOD: Handle Select all
+            }}
+          />
+        ),
+        accessor: row => row.kind,
+        width: '22%',
+        //TODO: Remove this once var is used
+        // eslint-disable-next-line no-empty-pattern
+        Cell: ({}: CellProps<Feature>) => {
```

Comment:
In the meantime, you could just remove the argument
```suggestion
        Cell: () => {
```

Reply:
done, thanks